_data/faq.yml

```diff
@@ -1254,7 +1254,72 @@
                 Es wird auf dem Bildschirm eine Schaltfläche angezeigt, die es dir erlaubt die Google Play Dienste zu installieren/aktualisieren. Dazu wird ein Google-Konto und Zugriff auf den Google Playstore benötigt.
                 
                 Wenn du dies nicht installieren kannst oder möchtest, kannst du stattdessen eine der OSM-Karten nutzen, die von c:geo unterstützt werden. Klicke hierzu einfach auf das Kartensymbol in der Titelzeile und wähle eine andere Karte anstatt Google aus.
+ 
+- id: ALC
+  title:
+    en: Adventure Lab Caches
+    de: Adventure Lab-Caches
+  items:
+      - id: ALC_PM
+        en:
+            title: Why are Adventure Lab Caches only shown to premium members?
+            content: |
+                There is no final information available from Groundspeak yet, how Adventure Labs will be implemented into their application interfaces and/or websites.
+                Therefore we decided for the time being to stick to a minimum approach, which is following the recommendation by Groundspeak and equals the implementation of other Geocaching apps.
+                
+                If we would include all currently available data for all member types, we might later end up with removing functions, if they are no longer available by Groundspeaks future interfaces.
+        de:
+            title: Warum werden Adventure Lab-Caches nur Premium-Mitgliedern angezeigt?
+            content: |
+                TBC
+      - id: ALC_found
+        en:
+            title: Adventure Labs I have already found are not shown as found!
+            content: |
+                The current implementation works without any authorization over a public interface. 
+                As your account is not involved here, we can currently not provide any user specific status (e.g. whether you already found that Adventure Lab cache).
                 
+                You can however mark Adventure Lab caches as "Found" locally in c:geo (Menu XYZ). Please be aware, that this "Found" marking is lost, if you delete the Adventure Lab cache from c:geo.
+        de:
+            title: Adventure Labs, die ich schon gefunden habe, werden nicht als gefunden angezeigt!
+            content: |
+                TBC
+      - id: ALC_stage
+        en:
+            title: Why are the stages of Adventure Labs not shown?
+            content: |
+                There is no final information available from Groundspeak yet, how Adventure Labs will be implemented into their application interfaces and/or websites.
+                Therefore we decided for the time being to stick to a minimum approach, which is following the recommendation bv Groundspeak and equals the implementation of other Geocaching apps.
```

Comment:
Same here

Reply:
Sound good, will take this. Thanks.

Reply:
"recommendation bv Groundspeak"

Typo: bv ⇾ by